index.html

```diff
@@ -89,48 +97,142 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    party: 'all'
   };
-
-  var svg;
-
+  var svg, x, y;
+  var margin = { top: 15, right: 15, bottom: 30, left:45 };
+  var width = 640 - margin.right -  margin.left;
+  var height = 400 - margin.top - margin.bottom;
   // FETCH DATA
-
-  function setup() {
+  d3.json('data.json', function (error, json){
+    // Call back function
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+
+  function setup( ) {
     // SVG
-
     svg = d3.select('#chart')
-      .append('svg');
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform','translate(' + margin.left + ',' + margin.top + ')');
+   
     // SCALES
-
+    x = d3.scale.linear()
+      .domain([d3.min(data,function (d){ return d.cfScore;}), Math.ceil(d3.max(data, function (d){ return d.cfScore; } )) ])
+        .range([margin.left,(margin.left + width)])
+        .nice();
+
+console.log(Math.ceil(d3.max(data, function (d){ return d.cfScore; } )))
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d){ return d.distPart; } ))
+      .range([(height-margin.top),0])
```

Comment:
Remember that we've already subtracted the top margin from the height when we first declared height on line 105. Same thing for the width on 127. We did that (and added it back in when we sized the SVG) precisely so we wouldn't have to adjust it here or subsequently. Perhaps fixing that will achieve what you were trying to do with `Math.ceil()`?